src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ServiceCategoryDTO.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ComplexityLevel
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceCategory
+import java.util.UUID
+
+data class ServiceCategoryDTO(
```

Comment:
this looks like a waste of time, but a subtle difference in sending this over the wire vs the entity object is that there is no 'created' field, which is not required by the client.

Reply:
I don't think it's a bad idea to decouple from the database entity as this might expand over time